package.json

```diff
@@ -16,14 +16,14 @@
   },
   "dependencies": {
     "@babel/compat-data": "^7.11.0",
-    "@bcrs-shared-components/bread-crumb": "1.0.2",
+    "@bcrs-shared-components/bread-crumb": "1.0.8",
     "@bcrs-shared-components/confirm-dialog": "1.1.0",
     "@bcrs-shared-components/corp-type-module": "1.0.6",
     "@bcrs-shared-components/court-order-poa": "1.0.14",
-    "@bcrs-shared-components/enums": "1.0.14",
-    "@bcrs-shared-components/interfaces": "1.0.37",
-    "@bcrs-shared-components/staff-comments": "1.2.0",
-    "@bcrs-shared-components/staff-payment": "1.1.0",
+    "@bcrs-shared-components/enums": "1.0.20",
+    "@bcrs-shared-components/interfaces": "1.0.40",
+    "@bcrs-shared-components/staff-comments": "1.2.6",
+    "@bcrs-shared-components/staff-payment": "2.0.10",
```

Comment:
Major change, testing this now.

Reply:
I did a Director change, paid with a routing slip.. seemed to work.

Reply:
Thanks for checking! You could also do a diff (via GH history) of v1 versus v2 and target what you're testing.

Usually a major version change means something will be incompatible.